var/spack/repos/builtin/packages/git/package.py

```diff
@@ -236,6 +238,16 @@ def filter_rt(self):
             # Don't link with -lrt; the system has no (and needs no) librt
             filter_file(r' -lrt$', '', 'Makefile')
 
+    @run_after('configure')
+    def make_opts(self):
+        self.build_targets=['NO_INSTALL_HARDLINKS=1',
+                            'ETC_GITATTRIBUTES=config',
+                            'ETC_GITCONFIG=config']
+        self.install_targets=['NO_INSTALL_HARDLINKS=1',
+                              'ETC_GITATTRIBUTES=config',
+                              'ETC_GITCONFIG=config',
+                              'install']
```

Comment:
These don't need to be inside of a method, they can just be global variables defined inside the class.